### PR TITLE
Update docs with webpack config custom plugin example

### PR DESCRIPTION
**Pull request by godleman**
### Requirements

Related Issue/RFC: n/a
Update docs with webpack config custom plugin example

### Description of the Change

Update docs with webpack config custom plugin example

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.
- [ ] I have added a changeset to my PR. See [**CONTRIBUTING**](https://github.com/10up/10up-toolkit/blob/develop/CONTRIBUTING.md) document for instructions

### CODE REVIEW

File: packages/toolkit/README.md [hunk 1]
```diff
@@ -499,9 +499,26 @@ const config = require('10up-toolkit/config/webpack.config.js');
...
+```javascript
+// webpack.config.js
+const config = require('10up-toolkit/config/webpack.config.js');
+const ProjectSpecificPlugin = require('project-specific-plugin');
+
+config.plugins.push(
+
+	// Append project specific plugin config.
+	new ProjectSpecificPlugin(),
+);
```
Comment (Antonio-Laguna) [inline packages/toolkit/README.md:516]: I reckon we should add a note about the order of plugins and a pointer to the file and perhaps even a quick demo on how to put it between two plugins.

What are your thoughts?

**Review by Antonio-Laguna — DISMISSED** [review]
@cldhmmr this is looking great! Thanks for adding this and also fixing the `defaultConfig` reference! 

I left a minor note

File: packages/toolkit/README.md (hunk 1 above)
Comment (godleman) [reply packages/toolkit/README.md:516]: On this point - as discussed plugin order is not respected by webpack, according to the author so we agreed to leave this for now.

**Review by nicholasio — DISMISSED** [review]
Thank you for your contribution.

**Review by nicholasio — APPROVED** [review]

**Pull request merged**